src/analytics/ValoraAnalytics.ts

```diff
@@ -71,7 +100,7 @@ const SEGMENT_OPTIONS: analytics.Configuration = {
 
 class ValoraAnalytics {
   sessionId: string = ''
-  deviceInfo: object = {}
```

Comment:
@kathaypacific this was our type before. it allowed a promise as a value with no compile time error.

Reply:
so the fact that the library was typed did not save us because we were just dumping a mix of promises and strings here